usmqe/gluster/gluster.py

```diff
@@ -131,6 +131,23 @@ def get_hosts_from_trusted_pool(self, host):
         LOGGER.debug("Hosts in trusted pool: %s" % hosts)
         return hosts
 
+    def get_host_states_from_trusted_pool(self, host):
+        """
+        Returns dictionary of hosts where key is hostname and value is host
+        status of connectivity from trusted pool with given hostname.
```

Comment:
Marking *host status* as `True` or `False` is slightly "unclear" - what it means (the comment is clear though, so it is probably ok)...

Reply:
Renamed `get_host_states_from_trusted_pool` to `get_cluster_hosts_connection_states`.